lib/ChainwebData/TxDetail.hs

```diff
@@ -49,6 +49,7 @@ data TxDetail = TxDetail
   , _txDetail_previousSteps :: Maybe [Text]
   , _txDetail_signers :: [Signer]
   , _txDetail_sigs :: [Sig]
+  , _txDetail_verifiers :: Maybe Value
```

Comment:
I think we can do better than this. These verifiers are conceptually similar to signers, so we probably need a `Verifier` data type and a database table to go with it.

Reply:
While we could indeed consider adding verifiers as a new column to the `transactions` table, I think there's good reason to keep it in a separate table like the `signers` table, because there can be multiple verifiers in a given transaction and if one wanted to index by, say, the verifier name, that would be impossible to do via the `transactions` table.